### ISSUE TITLE
New term `of`

### ISSUE THREAD

**Issue by denismaier** [citation-style-language/schema#410]
See: https://github.com/citation-style-language/styles/issues/5906

In some cases a new term `of` might make sense, e.g., to build constructs like "special issue of". Question is whether that works with non-european languages.

**Comment by bdarcus** [comment]
> Question is whether that works with non-European languages.

You know more about this than I, but I would guess that could be a major impediment, which would only be solved by doing the whole term, rather than components of it?

**Comment by georgd** [comment]
That could even be an issue with European languages that rely on casus suffixes where English uses prepositions. But the same is true for other prepositions that already exist in CSL (on, for example).

**Comment by denismaier** [comment]
> That could even be an issue with European languages that rely on casus suffixes where English uses prepositions. But the same is true for other prepositions that already exist in CSL (on, for example).

True, that's an issue. Any suggestions for improvements here? I was contemplating about a new form as a potential solution for this problem, think of `term="special-issue" form="of"`, but well, you might think that falls under the category "making things overly complicated just for the sake of it".

**Comment by cormacrelf** [comment]
The task appears to be to handle, concurrently/in the same style, languages that use `of` and languages that can only use 's-style agglutination like Korean's -의. I suspect nearly all languages with genitive casing on the owning noun have a less-stylish way to do this without piercing the veil of the journal's title. 

A trick might be to have both a possessive form of noun terms AND a possessive suffix aka 's term. A locale can then define one and blank out the other.

**Comment by georgd** [comment]
I don’t see a general solution to this issue. @cormacrelf ’s solution probably works well for some agglutinating languages (those that "simply" attach suffixes to the words) but already there, those with vowel harmony pose problems because the suffixes look slightly different according to the vowels in the word itself (Turkish, Hungarian, Finnish etc).

Languages with nominal flection systems are even less probable to work well here (e.g. slavonic like Russian). Here, not only suffixes are attached but the original word-ending might change as well.

So, a Citeproc would need to know which word needs to be changed int which way in a possessive case. This can be managed in a dictionary in locale files but means that the scope of such term is restricted to a limited vocabulary (e.g. the words "journal" and "yearbook").

Another issue: grammatical gender. Consider the German examples

```
Zeitschrift für Physik
Jahrbuch für Philosophie
```

with "special issue of" added:

```
Sonderausgabe der Zeitschrift für Physik
Sonderausgabe des Jahrbuchs für Philosophie
```

**Comment by bwiernik** [comment]
So it sounds like neither `of` nor `special-issue-of` is wholly satisfactory because of the gender issue

**Comment by georgd** [comment]
That’s right. However, CSL already defines a couple of prepositions which are more or less affected by the same issues, so one can consider it consistent to ignore these.

IMO, the only full solution would be to make citeprocs aware of grammar rules, which is fairly out of scope, i think.

For languages using postpositions instead of prepositions (e.g. Hindi, Japanese, Hungarian do so but I don’t know if all of them would use a postposition in this very case or rather a bare inflected case), a solution like the one @cormacrelf suggests could be helpful.
